styles/globals.css

```diff
@@ -365,6 +365,72 @@ blockquote {
   color: #4b5563;
 }
 
+/* Footer social hover effects */
+
+#twitter:hover,
+#twitter:focus {
+  transition: all 300ms;
+  transform: scale(1.05);
+  filter: brightness(1.1);
+  cursor: pointer;
+  color: white !important;
```

Comment:
We should be able to do some if not most of this with some Tailwind too.
Also, if possible remove the `!important`. The use of `!important` always causes bugs somewhere 😢